The LOCALBASE variable is used as a prefix for parts of both build and install.
This causes issues with cross-compiling, where the prefix for development files
and for installation are different (e.g. /usr/aarch64-linux-gnu versus /usr).

This patch removes references to LOCALBASE in the include and link search paths
to fix include errors caused by having `-I/usr/include` in a cross-compile.

--- a/Makefile.gnu	2022-07-25 15:21:33.867612285 -0500
+++ b/Makefile.gnu	2022-07-25 15:41:18.197622258 -0500
@@ -23,7 +23,6 @@
 CRYPT_LDADD=	-lcrypto
 endif
 
-CFLAGS+=	-I${LOCALBASE}/include
 CFLAGS+=	-Wall -I${.CURDIR}
 CFLAGS+=	-Wstrict-prototypes -Wmissing-prototypes
 CFLAGS+=	-Wmissing-declarations
@@ -32,7 +31,6 @@
 CFLAGS+=	${CRYPT_CFLAGS}
 CFLAGS+=	-I${CURDIR} -I${CURDIR}/openbsd-compat/
 
-LDFLAGS+=	-L${LOCALBASE}/lib
 LDFLAGS+=	${CRYPT_LDFLAGS}
 LDADD+=		${CRYPT_LDADD} -lopensmtpd
 
